File: output/pascal/suma_do_n.pas
program SumaDoN ;

var
    n : integer ;
    i : integer ;
    suma : integer ;

begin
    writeln ( "Podaj n: " ) ;
    readln ( n ) ;

    suma := 0 ;

    for i := 1 to n do
        suma := suma + i ;

    writeln ( "Suma od 1 do n: " ) ;
    writeln ( suma ) ;
end .

File: output/c/suma_do_n.c
#include <stdio.h>

/* Program: SumaDoN */
int main(void) {
    int n;
    int i;
    int suma;
    
    printf("Podaj n: ");
    scanf("%d", &n);
    suma = 0;
    for (i = 1; i <= n; i++) {
        suma = (suma + i);
    }
    printf("Suma od 1 do n: ");
    printf("%d\n", suma);
    return 0;
}
